turbopack/crates/turbopack-ecmascript/src/analyzer/graph.rs

```diff
@@ -1242,7 +1242,13 @@ impl VisitAstPath for Analyzer<'_> {
                 if let Some(require_var_id) = extract_var_from_umd_factory(callee, &n.args) {
                     self.add_value(
                         require_var_id,
-                        JsValue::WellKnownFunction(WellKnownFunctionKind::Require),
+                        JsValue::WellKnownFunction(WellKnownFunctionKind::Require {
+                            ignore: self
+                                .eval_context
+                                .imports
+                                .get_overrides(n.callee.span())
+                                .ignore,
+                        }),
                     );
```

Comment:
Why having `WellKnownFunctionKind ::Require { ignore: true }` at all? 

This could just be `JsValue::Unknown` for a ignored `require` or `import`.

Reply:
This might've been needed when this code wasn't bailing out as early before. I'll try removing it :+1: